dev-docs/source/environment.rst

```diff
@@ -47,6 +47,15 @@ This can be changed using::
     DSO_API_LOG_LEVEL = ...
     DSO_API_AUDIT_LOG_LEVEL = ...
 
+For some endpoints, when error messages are generated, those message
+are containing too much confidential information (like BSN numbers) to end up in the Sentry logs.
+
+So, it is possible to blacklist certain endpoints based on the `path` part of the url
+using the environment variable `SENTRY_BLOCKED_PATHS`. This environment variable
+if defined, should contain a comma-separated list of strings that will flag the endpoint for
```

Comment:
```suggestion
should contain a comma-separated list of strings that will flag the endpoint for
```

Either this, or add a comma before "if defined" (but less words is more better, right?).